Apsitvarkom.DataAccess/IGeocoder.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Apsitvarkom.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Apsitvarkom.DataAccess
+{
+    public interface IGeocoder
+    {
+        public Task<string> ReverseGeocode(Coordinates cords);
```

Comment:
we could leave it as is though, as all our written interfaces have the access modifier public.

Also, I believe it defaults to internal and not public, at least so is written [here](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers#other-types)

Reply:
I'm talking about the `public` in `public Task<string> ReverseGeocode(Coordinates cords)`, sorry if that wasn't clear. In the page you linked it says:

> Interface members are public by default because the purpose of an interface is to enable other types to access a class or struct.